benchmarks/pipe.py

```diff
@@ -335,20 +241,12 @@ def get_last_device(model):
             find_unused_parameters=False,
         )
 
+    # FIXME: Why do we need this? Can we use synthetic data instead?
```

Comment:
I think this is needed for multiprocess pipe so that fake data is sent to the model replicas except the first and the last replicas. I think a comment here will be helpful. A TODO perhaps can be added to figure out how to avoid the need of this in the future in the API level.

Reply:
Done.